trinity/common/config.py

```diff
@@ -412,6 +412,7 @@ class ModelConfig:
     critic_model_path: str = ""
 
     custom_chat_template: Optional[str] = None
+    chat_template_path: Optional[str] = None
```

Comment:
Add some comments here to describe how to use this field, and the difference with `custom_chat_tempate`